src/Common/ThreadFuzzer.cpp

```diff
@@ -31,6 +31,10 @@
         M(int, pthread_mutex_unlock, pthread_mutex_t * arg)
 #endif
 
+#if defined(__clang__) && __clang_major__ >= 13
```

Comment:
This fixes an error:
```
[65/2161] Building CXX object src/CMakeFiles/clickhouse_common_io.dir/Common/ThreadFuzzer.cpp.o
		FAILED: src/CMakeFiles/clickhouse_common_io.dir/Common/ThreadFuzzer.cpp.o 
		/usr/bin/clang++-13  -DAWS_SDK_VERSION_MAJOR=1 -DAWS_SDK_VERSION_MINOR=7 -DAWS_SDK_VERSION_PATCH=231 -DBOOST_ASIO_STANDALONE=1 -DENABLE_OPENSSL_ENCRYPTION -DPOCO_ENABLE_CPP11 -DPOCO_HAVE_FD_EPOLL -DPOCO_OS_FAMILY_UNIX -DSTD_EXCEPTION_HAS_STACK_TRACE=1 -DUNALIGNED_OK -DUSE_CPUID=1 -DUSE_JEMALLOC=1 -DUSE_REPLXX=1 -DWITH_COVERAGE=0 -DWITH_GZFILEOP -DX86_64 -DZLIB_COMPAT -I../contrib/sentry-native/include -Iincludes/configs -I../src -Isrc -Isrc/Core/include -I../base/glibc-compatibility/memcpy -I../base/widechar_width -I../base/common/.. -Ibase/common/.. -I../contrib/cityhash102/include -I../contrib/cctz/include -Icontrib/zlib-ng -I../contrib/zlib-ng -I../base/pcg-random/. -I../contrib/aws-c-common/include -I../contrib/aws-c-event-stream/include -Icontrib/aws-s3-cmake/include -Icontrib/bzip2 -isystem ../contrib/bzip2 -isystem ../contrib/fast_float/include -isystem ../contrib/msgpack-c/include -isystem ../contrib/brotli/c/include -isystem ../contrib/aws/aws-cpp-sdk-s3/include -isystem ../contrib/aws/aws-cpp-sdk-core/include -isystem ../contrib/xz/src/liblzma/api -isystem ../contrib/zstd/lib -isystem ../contrib/re2 -isystem ../contrib/libcxx/include -isystem ../contrib/libcxxabi/include -isystem ../contrib/libunwind/include -isystem ../contrib/jemalloc-cmake/include -isystem ../contrib/boost -isystem ../contrib/poco/Net/include -isystem ../contrib/poco/Foundation/include -isystem ../contrib/poco/NetSSL_OpenSSL/include -isystem ../contrib/poco/Crypto/include -isystem ../contrib/boringssl/include -isystem ../contrib/poco/Util/include -isystem ../contrib/poco/JSON/include -isystem ../contrib/poco/XML/include -isystem ../contrib/replxx/include -isystem ../contrib/fmtlib-cmake/../fmtlib/include -isystem ../contrib/double-conversion -isystem ../contrib/dragonbox/include -isystem contrib/re2_st -isystem ../contrib/libcpuid -isystem ../contrib/libc-headers/x86_64-linux-gnu -isystem ../contrib/libc-headers -fdiagnostics-color=always -fsized-deallocation  -gdwarf-aranges -pipe -mssse3 -msse4.1 -msse4.2 -mpclmul -mpopcnt -fasynchronous-unwind-tables -falign-functions=32   -Wall -Wno-unused-command-line-argument  -fdiagnostics-absolute-paths -fexperimental-new-pass-manager -Werror -Wextra -Wframe-larger-than=65536 -Wpedantic -Wno-vla-extension -Wno-zero-length-array -Wno-c11-extensions -Wcomma -Wconditional-uninitialized -Wcovered-switch-default -Wdeprecated -Wembedded-directive -Wempty-init-stmt -Wextra-semi-stmt -Wextra-semi -Wgnu-case-range -Winconsistent-missing-destructor-override -Wnewline-eof -Wold-style-cast -Wrange-loop-analysis -Wredundant-parens -Wreserved-id-macro -Wshadow-field -Wshadow-uncaptured-local -Wshadow -Wstring-plus-int -Wundef -Wunreachable-code-return -Wunreachable-code -Wunused-exception-parameter -Wunused-macros -Wunused-member-function -Wzero-as-null-pointer-constant -Weverything -Wno-c++98-compat-pedantic -Wno-c++98-compat -Wno-c99-extensions -Wno-conversion -Wno-ctad-maybe-unsupported -Wno-deprecated-dynamic-exception-spec -Wno-disabled-macro-expansion -Wno-documentation-unknown-command -Wno-double-promotion -Wno-exit-time-destructors -Wno-float-equal -Wno-global-constructors -Wno-missing-prototypes -Wno-missing-variable-declarations -Wno-nested-anon-types -Wno-packed -Wno-padded -Wno-shift-sign-overflow -Wno-sign-conversion -Wno-switch-enum -Wno-undefined-func-template -Wno-unused-template -Wno-vla -Wno-weak-template-vtables -Wno-weak-vtables -O2 -g -DNDEBUG -O3  -fno-pie   -D OS_LINUX -nostdinc++ -pthread -std=gnu++2a -MD -MT src/CMakeFiles/clickhouse_common_io.dir/Common/ThreadFuzzer.cpp.o -MF src/CMakeFiles/clickhouse_common_io.dir/Common/ThreadFuzzer.cpp.o.d -o src/CMakeFiles/clickhouse_common_io.dir/Common/ThreadFuzzer.cpp.o -c ../src/Common/ThreadFuzzer.cpp
		/home/semin-serg/ClickHouse/src/Common/ThreadFuzzer.cpp:181:9: error: identifier '__cpu' is reserved because it starts with '__' [-Werror,-Wreserved-identifier]
		        CPU_SET(migrate_to, &set);
		        ^
		/home/semin-serg/ClickHouse/contrib/libc-headers/sched.h:83:33: note: expanded from macro 'CPU_SET'
		# define CPU_SET(cpu, cpusetp)   __CPU_SET_S (cpu, sizeof (cpu_set_t), cpusetp)
		                                 ^
		/home/semin-serg/ClickHouse/contrib/libc-headers/x86_64-linux-gnu/bits/cpu-set.h:60:14: note: expanded from macro '__CPU_SET_S'
		   ({ size_t __cpu = (cpu);                                                   \
		             ^
		/home/semin-serg/ClickHouse/src/Common/ThreadFuzzer.cpp:268:27: error: identifier '__pthread_mutex_lock' is reserved because it starts with '__' [-Werror,-Wreserved-identifier]
		FOR_EACH_WRAPPED_FUNCTION(MAKE_WRAPPER)
		                          ^
		/home/semin-serg/ClickHouse/src/Common/ThreadFuzzer.cpp:268:27: error: identifier '__pthread_mutex_unlock' is reserved because it starts with '__' [-Werror,-Wreserved-identifier]
		3 errors generated.
```